service/src/main/resources/application.yml

```diff
@@ -0,0 +1,44 @@
+logging:
```

Comment:
I kind of like how @rtitle did this in the billing profile manager, where the environment variables are all at the top, so I copied that for catalog. Did you consider doing it that way and reject it? I don't really know the pros/cons of each way so I'd be interested to hear if there's a reason to prefer one way or the other.  https://github.com/DataBiosphere/terra-billing-profile-manager/blob/main/service/src/main/resources/application.yml